README.md

```diff
@@ -0,0 +1,31 @@
+# LSP-vtsls
+VSCode's TypeScript and JavaScript support for Sublime's LSP plugin provided through [vtsls](https://github.com/yioneko/vtsls).
+
+## What is vtsls ?
+This is an LSP wrapper around TypeScript extension bundled with VSCode. All features and performance are nearly the same. you can enjoy VSCode's typescirpt features into Sublime Text.
+
+## Installation
+
+ * Install [`LSP`](https://packagecontrol.io/packages/LSP) and `LSP-typescript` from Package Control.
+ * For ST4: The TypeScript and React (TSX) syntaxes are built-in so no need to install anything else.
+ * For ST3: If you are working with TypeScript install [TypeScript Syntax](https://packagecontrol.io/packages/TypeScript%20Syntax). If you are working with React install [JSCustom](https://packagecontrol.io/packages/JSCustom).
```

Comment:
You can remove this section `* For ST3: If you are working with TypeScript install [` As this package is no longer ST3 compatible.